contentScript.js

```diff
@@ -118,14 +118,14 @@
       itemList.appendChild(item);
       item.toolbar = toolbar;
       item.answer = answers[i].description;
-      item.addEventListener('click', insertAnswer);
+      $(item).on('selectmenu:selected', insertAnswer);
```

Comment:
Nope :(
<img width="674" alt="screen shot 2016-01-29 at 3 36 28 pm" src="https://cloud.githubusercontent.com/assets/1369170/12691155/17cd40d4-c69e-11e5-97b6-82fde1274cbf.png">

And `item.addEventListener('selectmenu:selected', insertAnswer);` doesn't seeeeem to work